src/test/java/com/rocketden/main/api/ProblemTests.java

```diff
@@ -39,16 +42,37 @@
     @Autowired
     private MockMvc mockMvc;
 
+    private static final String GET_PROBLEM = "/api/v1/problems/%s";
     private static final String GET_PROBLEM_ALL = "/api/v1/problems";
     private static final String POST_PROBLEM_CREATE = "/api/v1/problems";
+    private static final String POST_TEST_CASE_CREATE = "/api/v1/problems/%s/test-case";
 
     private static final String NAME = "Sort an Array";
     private static final String DESCRIPTION = "Sort an array from lowest to highest value.";
     private static final String NAME_2 = "Find Maximum";
     private static final String DESCRIPTION_2 = "Find the maximum value in an array.";
 
+    private static final String INPUT = "[1, 2, 8]";
+    private static final String OUTPUT = "8";
```

Comment:
Were you intending these `INPUT` / `OUTPUT` values to match the **first** `NAME` / `DESCRIPTION`?  In that case, you'd change the `OUTPUT` to `[1, 2, 8]`, otherwise no problem.

Reply:
Haha I wasn't but that's a good catch - updated in all the test files